### ISSUE TITLE
Player Name change issues

### ISSUE THREAD

**Issue by CruxCraft** [bloodmc/GriefDefender#169]
A player made a claim using OldName, then they changed to NewName... finding a lot of issues with listing and getting incorrect (if any) information about their claim

All done while standing in their claim:
Dynmap shows OldName
using a stick shows OldName
"/gd claim info" shows NewName
"/gd claim list NewName" shows no claims at all!
"/gd claim list OldName" says "Player OldName is not valid."
"/gd player info OldName" says "Player 'OldName' could not be found."
"/gd player info NewName" shows Total Claims: 1
"/gd delete claim" says "Are you sure you want to delete OldName's claim?"

GriefDefender 1.2.6
git-Paper-162
LuckPerms v5.0.130

**Comment by kencinder** [comment]
Their account UUID has changed, have you switched from OFFLINE mode to ONLINE mode or vice versa?

This would change player UUIDs, and has nothing to do with GriefDefender, you would need to manually edit claimdata to change oldUUID > newUUID.

**Comment by CruxCraft** [comment]
Some parts of GD show new names and some show old... GriefDefender recognized the new name as the owner of the plot with "/gd claim info" which shows the NewName (as does player info command), while most parts of still show the old name... I am also assuming it has removed the OldName from a lot of places within GD as it says the name is not valid and that it could not be found when using commands with the OldName...

The server has not changed online mode. Other things that rely on UUID work correctly such as LWC and in game achievements... It is a waterfall/bungee server in ONLINE mode with the paper/spigot servers being in OFFLINE mode. I would think this would be somewhat irrelevant considering GriefDefender IS recognizing the name change in some places it uses it, just still storing/displaying the old name in most places...

Also, i couldn't even see the claim at all with either name using /gd claim list <playerName> but I wouldnt be able to tell you if this is the case before the name change

I'd hoped that listing all the commands I tried and their results in the first comment would explain all that. Sorry if it made it more confusing... I wouldn't have even known except I noticed their claim on Dynmap. I hadnt even let them know we had GriefDefender yet so it was interesting to see a player had already made a claim... Just had to be one of the players that keeps changing their name...
